src/Symfony/Component/Console/composer.json

```diff
@@ -23,7 +23,8 @@
         "symfony/event-dispatcher": "~2.8|~3.0",
         "symfony/filesystem": "~2.8|~3.0",
         "symfony/process": "~2.8|~3.0",
-        "psr/log": "~1.0"
+        "psr/log": "~1.0",
+        "phpunit/phpunit": "~4.0"
```

Comment:
Not sure why that's showing, I reverted that change last night - was committed by mistake